ReactiveCocoa/UIKit/UIControl.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  UIView.swift
+//  Rex
+//
+//  Created by Neil Pankey on 6/19/15.
+//  Copyright (c) 2015 Neil Pankey. All rights reserved.
+//
+
+import ReactiveSwift
+import UIKit
+import enum Result.NoError
+
+private class UnsafeControlReceiver: NSObject {
+	private let observer: Observer<(), NoError>
+
+	fileprivate init(observer: Observer<(), NoError>) {
+		self.observer = observer
+	}
+
+	@objc fileprivate func sendNext() {
+		observer.send(value: ())
+	}
+}
+
+extension UIControl {
+	public func trigger(for events: UIControlEvents) -> Signal<(), NoError> {
+		return Signal { observer in
+			let receiver = UnsafeControlReceiver(observer: observer)
+			addTarget(receiver, action: #selector(UnsafeControlReceiver.sendNext), for: events)
+
+			let disposable = rac_lifetime.ended.observeCompleted(observer.sendCompleted)
+
+			return ActionDisposable { [weak self] in
+				disposable?.dispose()
+				self?.removeTarget(receiver, action: #selector(UnsafeControlReceiver.sendNext), for: events)
+			}
+		}
+	}
+
+	#if os(iOS)
+	/// Creates a bindable property to wrap a control's value.
+	///
+	/// This property uses `UIControlEvents.ValueChanged` and `UIControlEvents.EditingChanged`
+	/// events to detect changes and keep the value up-to-date.
+	//
+	class func rac_value<Host: UIControl, T>(_ host: Host, getter: @escaping (Host) -> T, setter: @escaping (Host, T) -> ()) -> MutableProperty<T> {
+		return associatedProperty(host, key: &valueChangedKey, initial: getter, setter: setter) { property in
+			property <~
+				host.trigger(for: [.valueChanged, .editingChanged])
+					.map { [unowned host] in getter(host) }
+		}
+	}
+	#endif
+
+	/// Wraps a control's `enabled` state in a bindable property.
+	public var rac_enabled: MutableProperty<Bool> {
+		return associatedProperty(self, key: &enabledKey, initial: { $0.isEnabled }, setter: { $0.isEnabled = $1 })
```

Comment:
`rac_enabled`, `rac_selected`, and any other properties like this seem like they should be either:
1) Made to use KVO for observing changes the happen via the original UIKit setter.
2) Left out of ReactiveCocoa until they do provide that two-way binding.
or 3) All be documented with a warning indicating that they're effectively write-only bindings.